pkgs/tools/misc/ddccontrol/default.nix

```diff
@@ -43,6 +43,7 @@ stdenv.mkDerivation rec {
     newPath="${ddccontrol-db}/share/ddccontrol-db"
     sed -i -e "s|$oldPath|$newPath|" configure.ac
     sed -i -e "s/chmod 4711/chmod 0711/" src/ddcpci/Makefile*
+    sed -i -e "s|/etc/|\$""{sysconfdir}/|" src/lib/Makefile*
```

Comment:
I guess this is because of [this change](https://github.com/ddccontrol/ddccontrol/compare/0.5.2...0.5.3#diff-29e335769d1450c75f2c4f3b1efc179ee7a17c6f075b407396a62b2667a738ac)?

But I am confused, we fetch the source from git and that rightfully does not contain Makefile, only the sources for Automake.

Also it might be better to use `substituteInPlace` as that will print a warning when the pattern no longer exists.

Finally, I believe files like these should not go to `/etc` at all but rather `${prefix}/lib`. Reading the appropriate [manpage](https://www.systutorials.com/docs/linux/man/5-modules-load.d/) confirms that so this is an upstream bug. (Though it does not particularly concern NixOS as we do not use these files.)

Reply:
Yes basically I wanted `make install` to work.
I changed the `sed` commands to `substituteInPlace` - good idea.

Reply:
Upstream PR: ddccontrol/ddccontrol#115